package.json

```
@@ -35,7 +34,8 @@
   },
   "lint-staged": {
     "*.js": "eslint --fix",
-    "types/**/*.ts": "prettier --write"
+    "*.ts": "eslint --fix",
```

```suggestion
    "*.{js,ts,tsx}": "eslint --fix",
```
then you can get rid of the others